ocaml-lsp-server/test/e2e/__tests__/textDocument-formatting.test.ts

```diff
@@ -32,6 +32,8 @@ function setupOcamlFormat(ocamlFormat: string) {
   return tmpdir;
 }
 
+const newline = process.platform == 'win32' ? "\r\n" : "\n";
```

Comment:
You're right, although I don't quite know how to mock an executable given the current config. As far as I know, Jest is great for mocking things inside a JS execution, and also for mocking files, but for executables I don't really know. I'll take a look at it, but if you have an idea, it would probably help :)